src/gap.c

```diff
@@ -791,6 +791,7 @@ Obj FuncWindowCmd (
   Int             n,  m;
   Int             i;
   Char *          ptr;
+  const Char *    inptr;
   Char *          qtr;
```

Comment:
Was there any reason/need to constify this? Looks like `SyWinCmd` returns a `Char*` anyway.

I only ask because this function doesn't seem to ever be called by anything I can see, so I'd be tempted to not touch it unless we have to (or of course remove it or test it..)